svc-aggregation/agmodel/system.go

```diff
@@ -143,6 +143,33 @@ func GetPluginData(pluginID string) (Plugin, *errors.Error) {
 	return plugin, nil
 }
 
+// CheckPluginConflictMgrAddr will check if manager address already exists
+func GetPluginFromMgrAddr(MgrAddr string) (Plugin, *errors.Error) {
```

Comment:
As discussed remove this function and use the existing model functions in the handlers.

Reply:
Done.